applicationconfig/client/src/main/java/com/azure/applicationconfig/ConfigurationAsyncClient.java

```diff
@@ -27,6 +27,73 @@
  * Client that contains all the operations for {@link ConfigurationSetting ConfigurationSettings} in Azure Configuration
  * Store.
  *
+ * <p>
+ *     Azure Configuration Store operations allowed by the client are adding, retrieving, updating, and deleting
+ *     ConfigurationSettings, additionally more complex operations of listing settings or revision of a setting based
```

Comment:
Adding a `{@link ConfigurationSetting ConfigurationSettings}` would make it easier to click on things and figure out what they are as I'm reading the docs.

Reply:
Do we want to @link all class and method references or are you asking for this particular spot?

Reply:
Do we want to `@link` all class and method references or do you want this particular instance linked?